pkg/storer/netstore.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	directUploadWorkers = 16
```

Comment:
This is not the same. We can have multiple calls to DirectUpload. It creates 16 workers for each upload.

Reply:
Does 16 workers mean 16 chunk push attempts in parallel?

Reply:
Yes 16 push attempts in parallel per upload. So the call to DirectUpload returns a putter which can use upto 16 routines to push in parallel.

Reply:
in that case, we do not actually need this limiter. the pusher is already limited.